assembl/views/auth/views.py

```diff
@@ -1119,6 +1120,20 @@ def send_confirmation_email(request, email, immediate=False):
         mailer.send(message)
 
 
+def maybe_send_email(request, profile, email=None,
+        subject=None, text_body=None, html_body=None,
+        discussion=None, sender_name=None, welcome=False,
+        immediate=False):
+        if profile:
+            send_change_password_email(request, profile, email,
+            subject, text_body=text_body, html_body=html_body, discussion=discussion,
+            sender_name=sender_name, welcome=welcome, immediate=immediate)
+            log.info('An email has been sent to the profile %s' %  email)
+        else:
+            log.error("This profile does not exist")
```

Comment:
Also, you should definitely pass the identifier to this function, not just the profile, so that your error log can be more clear.